assets/html/donate.html

```diff
@@ -401,6 +418,11 @@ <h1 id="how-much-title" class="disabled" tabindex="0">Quanto vuoi donare?</h1>
 <script>
   let allCharities = "";
 
+  const config = {
+    apiUrl: "https://api.uat.platform.pagopa.it/donations/api/v1/availabledonations",
```

Comment:
```suggestion
    apiUrl: "http://localhost:3000/donations/availabledonations",
```
I believe we should avoid any external dependency in dev-server